zerver/lib/user_groups.py

```diff
@@ -48,8 +83,16 @@ def user_groups_in_realm_serialized(realm: Realm) -> List[Dict[str, Any]]:
     )
     for (user_group_id, user_profile_id) in membership:
         group_dicts[user_group_id]["members"].append(user_profile_id)
+
+    group_membership = GroupGroupMembership.objects.filter(subgroup__realm=realm).values_list(
+        "subgroup_id", "supergroup_id"
+    )
+    for (subgroup_id, supergroup_id) in group_membership:
+        group_dicts[supergroup_id]["subgroups"].append(subgroup_id)
```

Comment:
I'm not sure if there's a way to do this, but ideally we'd do some sort of `select_related` thing that would let us do `UserGroup.objects.select_related("subgroups", "members").filter(realm=realm)` rather than doing 3 database queries here.